charts/vald/templates/gateway/backup/ing.yaml

```diff
@@ -31,11 +31,10 @@ metadata:
     app.kubernetes.io/component: gateway-backup
   name: {{ $gateway.name }}-ingress
 spec:
+  defaultBackend:
+    service:
+      name: {{ $gateway.name }}
+      {{- include "vald.ingressPort" (dict "Values" $gateway.ingress) | nindent 6 }}
```

Comment:
I think it should not be in the `defaultBackend` object.

> DefaultBackend is the backend that should handle requests that don't match any rule.

I'm not sure but it should be moved into `rules`? Otherwise, maybe we cannot have multiple ingresses in a single cluster.

Reply:
I know about the use of DefaultBackend, but when using the original(master branch) rules.http.paths.backend, networking.k8s.io/v1 now requires path and pathType field, and since we didn't use pathbased routing in this case, we're currently using single host In this case, the routing was not used in path, so now we should use DefaultBackend for single-host Ingress.

Reply:
I've just tested on my kubernetes cluster with Traefik Ingress controller, it doesn't work when only using defaultBackend like this branch. 
It works when using a yaml like below.
```yaml
spec:
  rules:
  - host: xxx.yyy.zzz #dummy
    http:
      paths:
      - backend:
          service:
            name: vald-gateway
            port:
              name: grpc
        pathType: ImplementationSpecific
```
As a result, at least the case of Traefik Ingress controller, pathType = ImplementationSpecific without defaultBackend is correct configuration. The chart should be revised. (Sorry I don't know about other Ingress controller)

Reply:
In my envitonment it's works...
I think it's coused on Traefik ??
https://kubernetes.io/docs/concepts/services-networking/ingress/#types-of-ingress

How do you think about pathType, ImplementationSpecific is not common, but we use gRPC each gateway has same prefix, I think it would be better to set Exact matching for PathType.

Reply:
it's confusing... :sweat_smile:

Reply:
are you using NGINX ingress controller?

Reply:
yes, and I tried contour....

Reply:
but Vald should be work on any environment.
we should fix it to the rules base routing.

Reply:
I'm not sure which vhost should we set on Ingress?

Reply:
as you defined as dummy.

Reply:
i set the VM host of the corp internal VM to vhost.

Reply:
you mean specific node's host??
or dns name??

Reply:
my cluster is a single node cluster, so i set the DNS name of the node host in my test.

Reply:
fixed thx!